### PR TITLE
Remove unneeded imports and old_div

### PR DESCRIPTION
**Pull request by aragilar**
This starts removing old_div. We want to remove old_div, replacing it with either `/` for normal division, or `//` for floor division. Feel free to keep adding to this PR.

### CODE REVIEW

File: src/koala/utils/plots.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
-from __future__ import division
+from __future__ import absolute_import, division, print_function
 from builtins import str
-from builtins import range
```
Comment (YourLocalBlake) [inline src/koala/utils/plots.py]: range is used in several places as `range(len(...))`, does this produce the same functionality in python 2.7 vs 3?

File: src/koala/utils/io.py [hunk 2]
```diff
@@ -1,35 +1,13 @@
 # -*- coding: utf-8 -*-
-from __future__ import division
-from __future__ import print_function
+from __future__ import absolute_import, division, print_function
+
 from builtins import str
-from builtins import range
```
Comment (YourLocalBlake) [inline src/koala/utils/io.py]: same comment as in plots.py, range is used several times

**Comment by YourLocalBlake** [comment]
`range()` is slightly different in 2.7 vs 3.x and am wondering if while python 3 work if all use cases have the same result - I'm not well enough educated in python 2.7

File: src/koala/utils/plots.py (hunk 1 above)
Comment (aragilar) [reply src/koala/utils/plots.py]: The `builtins` module that `futurize` uses makes `range` behave like python 3 on python 2. As long as we don't need range to produce a list (and wrap it when we do), range is fine on either version.

Comment (YourLocalBlake) [reply src/koala/utils/plots.py]: ah alright

**Pull request merged**